go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.12.1
 	github.com/submariner-io/admiral v0.12.0-m3.0.20220223161649-65232cddf5c0
-	github.com/submariner-io/shipyard v0.12.0-m3.0.20220421170648-adf7571e55ed
+	github.com/submariner-io/shipyard v0.12.0-m3.0.20220427131749-6b99f918e45e
```

Comment:
No need to do this commit, the image is available through Quay and you're just using it